onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -556,6 +557,13 @@ def _where_input_wrangler(
         matcher=lambda sample: len(sample.args[0]) == 0,
         reason="Empty perm is not supported",
     ),
+    skip(
+        "squeeze",
+        matcher=lambda sample: len(sample.args) > 0
+        and len(sample.input.shape) > 0
+        and sample.input.shape[sample.args[0]] != 1,
+        reason="Cannot select an axis to squeeze out which has size not equal to one",
```

Comment:
```suggestion
        reason="Cannot squeeze a dimension which has a value different than 1",
```

Reply:
the warning message is coping from onnxruntime. :)